source/agora/node/Ledger.d

```diff
@@ -1033,6 +1116,65 @@ unittest
     genBlockTransactionsFreeze(1, TxType.Freeze);
     assert(ledger.getBlockHeight() == 3);
 
+    // Create enrollments and add it in The block.
```

Comment:
Same comment here about testing multitude of features in the same unittest.

Reply:
It has been processed as above.